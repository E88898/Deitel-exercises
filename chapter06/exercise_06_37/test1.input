4
5
6
18
n
