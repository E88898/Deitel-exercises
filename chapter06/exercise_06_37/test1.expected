How much is 3 times 6? Wrong. Try once more. No. Keep trying. Wrong. Try once more. Keep up the good work!

Please ask your instructor for extra help
